R/test-elements.R

```diff
@@ -1495,7 +1498,10 @@ while_loop <- function(test, logic) {
       if (!is.scalar.logical(res)) stop("<test> did not return a ",
                                         "logical scalar")
       if ((skip_when == "pass" && res) ||
-          (skip_when == "fail" && !res)) skip_n_pages(state, skip_len)
+          (skip_when == "fail" && !res)){
+        message(sprintf("while_loop: Skipping %d pages (skip_when = %s)", skip_len, skip_when))
```

Comment:
Do you want to keep this in, or can it be removed now?

Reply:
Can be removed